pgwatch2/metrics/replication/10/metric_master.sql

```diff
@@ -1,7 +1,13 @@
 SELECT
   (extract(epoch from now()) * 1e9)::int8 as epoch_ns,
   application_name as tag_application_name,
+  *,
```

Comment:
Why do we need to add a star here? I suppose we should definitely list all needed columns instead